packages/components-js/tests/vrt/playwright/specs/common/components.vrt.ts

```diff
@@ -21,8 +21,12 @@ const components = (TAG_NAMES as unknown as TagName[])
 
 const isComponentThemeable = (component: string): boolean => getComponentMeta(`p-${component}` as TagName).isThemeable;
 
+// TODO: focus can't be reverted for flyout-navigation since it's rendered within an iframe and we don't have the possibility to evaluate when componentsReady() is resolved for all iframes used
+const revertAutoFocus = (component: string): boolean =>
+  ['banner', 'modal', 'flyout', 'flyout-navigation'].includes(component);
```

Comment:
This should be ideally configured in component meta like `isComponentThemeable`

Reply:
I've added an issue for this, #2923